src/xmipp/libraries/reconstruction/volume_to_pseudoatoms.cpp

```diff
@@ -935,21 +935,43 @@ void ProgVolumeToPseudoatoms::writeResults()
         if (allowIntensity)
             intensity=0.01+ROUND(100*a*(atoms[n].intensity-minIntensity))/100.0;
         if (col==1)
-            fprintf(fhOut,
-                    "ATOM  %5d DENS DENS%5d    %8.3f%8.3f%8.3f%6.2f     1      DENS\n",
-                    n+1,n+1,
-                    (float)(atoms[n].location(2)*sampling),
-                    (float)(atoms[n].location(1)*sampling),
-                    (float)(atoms[n].location(0)*sampling),
-                    (float)intensity);
+            if (n+1 < 10000)
```

Comment:
Why? You can print both with `%5d` and it will be justified to right with no leading zeros.
If you need justification to right, you can use `%-5d`

Reply:
ok. I'll try that

Reply:
oh, right. The reason is I wanted to use the 5th column only if I need it. Probably, that's unnecessary complexity though.

Reply:
I don't understand. 
The formatting needs to follow the standard, what does the standard say about the length?